UPDATING.md

```diff
@@ -1773,6 +1766,69 @@ Now the `dag_id` will not appear repeated in the payload, and the response forma
 }
 ```
 
+## Airflow 1.10.14
+
+### `[scheduler] max_threads` config has been renamed to `[scheduler] parsing_processes`
+
+From Airflow 1.10.14, `max_threads` config under `[scheduler]` section has been renamed to `parsing_processes`.
+
+This is to align the name with the actual code where the Scheduler launches the number of processes defined by
+`[scheduler] parsing_processes` to parse the DAG files.
+
+### Airflow CLI changes in line with 2.0
+
+The Airflow CLI has been organized so that related commands are grouped together as subcommands,
+which means that if you use these commands in your scripts, you have to make changes to them.
```

Comment:
Description here is a bit inaccurate.

I understand this part is referred from https://github.com/apache/airflow/blob/master/docs/apache-airflow/upgrading-to-2.rst  but a difference here is: in 1.10.14, users will get a deprecation warning if they use the "old" commands, but they don't get a hard failure (they don't `have to make changes` in 1.10.14).

Reply:
Suggest to change to

_The Airflow CLI in 1.10.14 has been changed to be in line with CLI in 2.0. The 1.10.* style CLI will still work, but users will get deprecation warning messages to be reminded to upgrade their scripts. This is is part of 1.10.14's 'bridge' role._